operator/cmd/mesh/manifest-generate.go

```diff
@@ -99,6 +99,15 @@ func manifestGenerate(args *rootArgs, mgArgs *manifestGenerateArgs, logopts *log
 		return fmt.Errorf("could not configure logs: %s", err)
 	}
 
+	inFileName := make([]string, 0)
+	for _, p := range mgArgs.inFilename {
+		if strings.Contains(p, fmt.Sprintf("%s.yaml", name.DefaultProfileName)) {
```

Comment:
This seems like a fragile workaround. I could have a file `default.yaml` that has a ton of non-default customizations

Reply:
@howardjohn Agree. Please take a look at the newer revision. Thanks!